cmd/wasm/main.go

```diff
@@ -0,0 +1,25 @@
+//go:build js && wasm
+
+package main
+
+import (
+	"fmt"
+	"syscall/js"
+)
+
+func main() {
+	// Create a channel to keep the Go program alive
+	done := make(chan struct{}, 0)
```

Comment:
```suggestion
	done := make(chan struct{})
```

it's fine for unbuffered to be implied imo